Books/client/src/features/misc/routes/UserLibrary/UserLibrary.tsx

```diff
@@ -1,9 +1,9 @@
-import React, { useEffect, useState } from "react";
-import { useOutletContext } from "react-router-dom";
+import React, { useContext, useEffect, useState } from "react";
 import "./UserLibrary.scss";
 import { Navigation } from "swiper";
 import { Swiper, SwiperSlide } from "swiper/react";
 
+import { AuthContext } from "../../../..//context/auth.context";
```

Comment:
Why are there two forward slashes for the path? I suspect this is a typo but it could easily get confusing for other people

Reply:
Sorry, you right, just a typo